### PR TITLE
Add CLI log writer (FR 4830)

### PR DESCRIPTION
**Pull request by WinterSilence**
Feature Request  http://dev.kohanaframework.org/issues/4830

### CODE REVIEW

File: classes/Kohana/Log/CLI.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	 *     $writer->write($messages);
+	 *
+	 * @param   array  $messages
+	 * @return  void
+	 */
+	public function write(array $messages)
+	{
+		foreach ($messages as $message)
+		{
+			$stream = ( ! isset($message['level']) OR $message['level'] < Log::NOTICE) ? STDERR : STDOUT;
```
Comment (kemo) [inline classes/Kohana/Log/CLI.php]: When is the message level not set? (@ isset check)

Comment (WinterSilence) [inline classes/Kohana/Log/CLI.php]: it's public method

```
(new Log_CLI)->write(array('foo' => 'bar'));
```

Comment (kemo) [inline classes/Kohana/Log/CLI.php]: Yes it's public but the format is well known and relied on, I'd remove the isset() check. Kohana was always known for being strict when it comes to things like this. 

Anyone else, opinions?

Comment (shadowhand) [inline classes/Kohana/Log/CLI.php]: agree, there is no reason to second guess a known format.

Comment (WinterSilence) [inline classes/Kohana/Log/CLI.php]: Function needs to check the input data. If Kohana uses some simplifications, it does not mean that this is the right approach.

**Comment by kemo** [comment]
I merged new commits manually, you don't need to open a new PR. Thanks!

**Pull request merged**